File: Exam 4 - Hackathon/exam9.js
// Task 9: Check if a Number is Positive
// Instructions:
// Write a function isPositive(num) that returns true if the number is positive, otherwise false.
// Test Cases:
// console.log(isPositive(10)); // true
// console.log(isPositive(-5)); // false
// console.log(isPositive(0)); // false
// console.log(isPositive(100)); // true
// console.log(isPositive(-1)); // false

function isPositive(num){
    if (num > 0){
        return true
    } else{
        return false
    }
}

console.log(isPositive(0))

File: Exam 4 - Hackathon/exam9.py
# Task 9: Factorial Calculation
# Instructions:
# Write a function factorial(n) that takes a non-negative integer and returns its factorial. The function should:
# Accept an integer as input.
# Return the factorial of the number.
# Handle the case where n = 0, which has a factorial of 1.
# Test Cases:
# assert factorial(0) == 1 
# assert factorial(1) == 1 
# assert factorial(5) == 120 
# assert factorial(7) == 5040 
# assert factorial(3) == 6

def factorial(n):
    num = 1
    for i in range(n):
        num *= i + 1
    return num
print(factorial(5))
